docs/website/blog/2022-01-14-odo-v2.5.0.md

```diff
@@ -13,22 +13,23 @@ slug: odo-250-release
 
 ## Notable changes in odo 2.5.0
 
-
 ### Support for ephemeral field in Devfile `volumes` definition ([#5279](https://github.com/redhat-developer/odo/pull/5279) [@feloy](https://github.com/feloy))
+
 odo now supports Devfile volumes that are defined as ephemeral. If volume is defined with `ephemeral: true` odo will create it as Kubernetes `emptyDir` volume.
 
 Example of an ephemeral volume definition inside `devfile.yaml`
+
 ```yaml
 - name: volume-test
   volume:
     size: 1Gi
     ephemeral: true
 ```
 
-
 ### Delete outerloop resources with `odo delete --deploy`([PR#5276](https://github.com/redhat-developer/odo/pull/5276) [@valaparthvi](https://github.com/valaparthvi))
-Now you can delete resoures that were deployed using `odo deploy` command using `odo delete` command with `--deploy` flag.
-`odo delete --all` now deletes everything from the cluster related to give Devfile incudling outerloop resources.
+
+Now you can delete resources that were deployed using `odo deploy` command using `odo delete` command with `--deploy` flag.
+`odo delete --all` now deletes everything from the cluster related to the given Devfile including outer-loop resources.
```

Comment:
is it outerloop or outer-loop?

Reply:
according to google it looks like Outerloop = a company, outer-loop = seldom used, outer loop seems to be the most correct